### PR TITLE
Split off external services

### PR DESCRIPTION
**Pull request by maximiliankolb**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)

### CODE REVIEW

**Review by ekohl — CHANGES_REQUESTED** [review]
Overall I'm still in favor of a DHCP guide and DNS guide. They should explain what the DHCP model in Foreman is and how the integration can happen. The same with DNS. That means not just describing how you set up the server, but also configure the domain(s). External services is just vague.

I would also extract parts from the provisioning guide (https://docs.theforeman.org/nightly/Provisioning_Hosts/index-katello.html#Using_Infoblox_as_DHCP_and_DNS_Providers_provisioning) and direct the reader to the external services guide to set up DHCP and/or DNS. There's also https://docs.theforeman.org/nightly/Provisioning_Hosts/index-katello.html#Configuring_Network_Services_provisioning.

Put in a different way: there are multiple ways to talk to an external DNS server.

One way is using the `nsupdate` provider to talk to a [RFC 2136](https://datatracker.ietf.org/doc/html/rfc2136) compatible DNS server. For that there are some ways to authenticate it. TSIG-GSS is a more complex way.

Then you can also install the Infoblox DNS plugin to talk to a remote DNS server, using the Infoblox API. In upstream we also have PowerDNS and Route 53 plugins.

So depending on how you count it, there are already 4 or 5 different DNS integrations alone. I'd argue that's worth its own DNS guide.

For DHCP the story is similar, but a bit shorter.

First there is the ISC DHCP integration that we recommend by default. There's a remote flavor as well, using NFS to monitor the files. And the third option is the Infoblox DHCP plugin. It may be shorter, but it's a complex topic.

**Comment by maximiliankolb** [comment]
@maximiliankolb  please rebase and add introduction to external services.

File: guides/common/attributes-titles.adoc [hunk 1]
```diff
@@ -3,6 +3,7 @@
 :AdministeringDocTitle: Administering {ProjectName}
 :AdministeringAnsibleDocTitle: Managing {Project} with Ansible
 :AppCentricDeploymentDocTitle: Deploying Hosts using Application Centric Approach
+:ConfiguringExternalServicesDocTitle: Configuring External Services
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/attributes-titles.adoc]: Applied.

File: guides/doc-Configuring_External_Services/docinfo.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<title>Configuring External Services</title>
+<productname>Red Hat Satellite</productname>
+<productnumber>6.15</productnumber>
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/doc-Configuring_External_Services/docinfo.xml]: Yes, applied.

File: guides/doc-Installing_Proxy/master.adoc [hunk 3]
```diff
@@ -28,7 +28,7 @@ include::common/assembly_installing-capsule-server.adoc[leveloffset=+1]
 include::common/assembly_performing-additional-configuration-on-capsule-server.adoc[leveloffset=+1]
 
 // Configuring {SmartProxyServer} with External Services
-include::common/assembly_configuring-external-services.adoc[leveloffset=+1]
+// -> See "Configuring External Services
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/doc-Installing_Proxy/master.adoc]: No, this was just a hint to myself. Dropped.

File: guides/doc-Configuring_External_Services/docinfo.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<title>{ConfiguringExternalServicesDocTitle}</title>
+<productname>Red Hat Satellite</productname>
+<productnumber>{ProjectVersion}</productnumber>
+<subtitle>TODO</subtitle>
+<abstract><para>TODO</para></abstract>
```
Comment (maximiliankolb) [inline guides/doc-Configuring_External_Services/docinfo.xml:5]: Kindly asking for input here.

**Comment by maximiliankolb** [comment]
reconsider "external services". maybe use "services" as category including DHCP/DNS/TFTP, irrespective if internal or external.

for the user: "how do I integrate X into Foreman?"

See also https://github.com/theforeman/foreman-documentation/issues/2957

**Comment by maximiliankolb** [comment]
triage: closed due to inactivity.

**Pull request closed without merging**